frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-install-global.spec.ts

```diff
@@ -17,23 +18,42 @@ xdescribe(`Interacting with a global install mode Operator (${operatorName})`, (
     cy.logout();
   });
 
-  const operatorPkgName = 'portworx-essentials';
-  const operatorID = 'portworx-essentials-community-operators-openshift-marketplace';
+  const catalogSourceName = 'console-e2e';
+  const operatorID = 'portworx-essentials-console-e2e-openshift-marketplace';
   const operatorRow = 'Portworx Essentials';
-  const operatorInstallFormURL = `/operatorhub/subscribe?pkg=${operatorPkgName}&catalog=community-operators&catalogNamespace=openshift-marketplace&targetNamespace=`;
+  const operatorPkgName = 'portworx-essentials';
+  const operatorInstallFormURL = `/operatorhub/subscribe?pkg=${operatorPkgName}&catalog=${catalogSourceName}&catalogNamespace=openshift-marketplace&targetNamespace=`;
   const operatorInstance = 'StorageCluster';
   const openshiftOperatorsNS = 'openshift-operators';
   const operandLink = 'portworx';
 
+  it('installs the catalogsource', () => {
+    cy.log('navigate to OperatorHub > Sources');
+    cy.visit('/k8s/cluster/config.openshift.io~v1~OperatorHub/cluster/sources');
+    cy.log('navigate to the CatalogSource creation form');
+    cy.byTestID('item-create').click();
+    cy.byTestID('create-catalogsource-title').should('exist');
+    cy.byTestID('catalog-source-name').type(catalogSourceName);
+    cy.byTestID('catalog-source-image').type(
+      'quay.io/operator-framework/upstream-community-operators@sha256:9243fc9cfdbe3df502772b92abdb0b4794adb4898833fb0a8a5c2f34f6e7ff4e',
+    );
+    cy.byTestID('save-changes').click();
+    cy.log('verify the catalogsource is READY');
+    cy.visit('/k8s/cluster/config.openshift.io~v1~OperatorHub/cluster/sources');
+    cy.byLegacyTestID(catalogSourceName).should('exist');
+    cy.byTestID(`${catalogSourceName}-status`).should('contain', 'READY');
+    // cy.wait(5000);
```

Comment:
I thought I could avoid using the wait by navigating to the packagemanifest list and confirming it exists (lines 46-49), but even that happens too quickly.  @dtaylor113, any better ideas on how to wait for the packagemanifests to be ready?

Reply:
Hi @rhamilto, can you do some `oc cli` [checking](https://github.com/openshift/console/blob/66a08e0c120137138888ca879cd7cd7bd70b1b75/frontend/packages/integration-tests-cypress/support/resources.ts#L33) to see if packagemanifests exists?   Can you [wait for a promise](https://docs.cypress.io/api/utilities/promise.html#Waiting-for-Promises)?  If not, you might have to do the `cy.wait`.  After the update to Cypress 6, we can [wait on a request](https://docs.cypress.io/api/commands/intercept.html#Waiting-on-a-request), but not sure that would help in this case?